Add credits screen and reorganize sprites, #98

AxlLind opened the pull request:
Add a suggested design for the credits screen. I've added it to the
start screen now but we might want to move it to the settings screen,
thoughts? Not sure which is better.

Also some reorganization of the sprites, that's why there are so many changed files in this pull request. No major code added.

AxlLind committed:
Add credits screen and reorganize sprites, #98

Add a suggested design for the credits screen. I've added it to the
start screen now but we might also move it to the settings screen,
thought?

Also some reorganization of the sprites.

Jontpan reviewed (changes requested):
The credits screen looks totally sick! I think it should be under settings though 👍

AxlLind committed:
Moved the credits screen to settings

AxlLind committed:
Add function to credits-button

Forgot to link the function to the button on the settings menu which
takes you to the credits.

AxlLind commented:
Credits now moved to the settings screen. Will merge!
